### ISSUE TITLE
More control over output - padding

### ISSUE THREAD

**Issue by marss72** [spectreconsole/spectre.console#1542]
**Is your feature request related to a problem? Please describe.**
Yes. I love having some padding on the left when building a Console UI (not cli tools, but interactive app). There is no solution (or at least I don't know about it)

**Describe the solution you'd like**
Add padding for more types, like spinners and much more. So like spinners would have a something like IRenderable (something similar), that would contain config how the spinner should be rendered (padding, position, ...)

**Describe alternatives you've considered**
Creating a new spinner with two space padding on the left for every frame. This is possible but not clean and should get implemented into this library.

**Additional context**
None.

---
Please upvote :+1: this issue if you are interested in it.

**Comment by patriksvensson** [comment]
Have you seen the `Padder` widget? https://spectreconsole.net/api/spectre.console/padder/

**Comment by marss72** [comment]
> Have you seen the `Padder` widget? https://spectreconsole.net/api/spectre.console/padder/

Yes, but this doesn't work for things like status, or at least I don't know how to implement it properly.
